src/index.html

```diff
@@ -13,6 +13,38 @@
     />
```

Comment:
The `href` attribute in the link tag is pointing to an SCSS file (`styles/index.scss`). Browsers cannot interpret SCSS directly, so you need to compile your SCSS into a CSS file and link to the resulting CSS file instead.